src/api/getData.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto2";
+
+package api;
+
+message AppData {
+        optional bool motion = 1;
+        optional uint32 water = 2;
+}
```

Comment:
`AppData` is used for downlink; the app is not going to tell these values. These are typical downlink fields, like lights to be turned on, digitals to be put high etc